code/modules/mob/living/holders.dm

```diff
@@ -106,7 +106,9 @@
 
 		throw_range = 6 - w_class
 
-		if(w_class > W_CLASS_SMALL)
+		if(w_class > W_CLASS_SMALL && M.can_wear_as_hat)
```

Comment:
I don't understand the fix. Does `MUSTTWOHAND` somehow prevent the holder from being worn as a hat? Isn't `slot_flags = SLOT_HEAD` sufficient to let one wear the holder as a hat, so that you wouldn't need this new `can_wear_as_hat` var?

Reply:
Yes, there is a check in the code if an object is actively being two-handed, then it can not be equipped a throws an error.  This fox allows for the pillow bug to be unweilded from the twohand state so that it may be equipped.  The SLOT_HEAD flag is not sufficient for the any object that forces you to two hand it auch as these small mobs.

Reply:
can't you just check if the thing is a mob holder instead?

Reply:
Honestly I'd just remove the requirement to unwield things before equipping them or putting them in storage
That's probably more complicated than it sounds, but it can't be *too* hard

Reply:
I thought about it, but I was afraid of breaking something down the line and figured there had to be a reason for such to exist.  I think it had to do with 2handed weapons like the fire axe or double-bladed energy sword unintentionally being able to be stored while active otherwise and the off-hand placeholder object going crazy.